src/components/clusterprovider/azure/azureclusterprovider.ts

```diff
@@ -283,6 +284,8 @@ async function createCluster(previousData: any, context: azure.Context): Promise
         } };
     const createResult = await azure.createCluster(context, options);
 
+    reporter.sendTelemetryEvent("createCluster", { result: createResult.result.succeeded ? "success" : "fail", clusterType: previousData.clusterType });
```

Comment:
The disclosure statement in the README needs to be updated to reflect the additional telemetry.

Reply:
Yes，the readme has been updated